docs/REST API/rest-api.md

```diff
@@ -8744,10 +8749,10 @@ _Available in Fleet Premium_
 | &nbsp;&nbsp;&nbsp;&nbsp;deadline_days                   | integer | body | Hosts that belong to this team and are enrolled into Fleet's MDM will have this number of days before updates are installed on Windows.                                                                   |
 | &nbsp;&nbsp;&nbsp;&nbsp;grace_period_days               | integer | body | Hosts that belong to this team and are enrolled into Fleet's MDM will have this number of days before Windows restarts to install updates.                                                                    |
 | &nbsp;&nbsp;macos_settings                              | object  | body | macOS-specific settings.                                                                                                                                                                                  |
-| &nbsp;&nbsp;&nbsp;&nbsp;custom_settings                 | list    | body | The list of objects where each object includes .mobileconfig or JSON file (configuration profile) and label name to apply to macOS hosts that belong to this team and are members of the specified label.                                                                                                                                        |
+| &nbsp;&nbsp;&nbsp;&nbsp;custom_settings                 | list    | body | The list of objects where each object includes .mobileconfig or JSON file (configuration profile) and label name to apply to macOS hosts that are assigned to this team.                                                                                                                                        |
```

Comment:
@noahtalerman  After verification, this should be removed from the docs - the `Modify team` endpoint does not support setting the MacOS Settings (nor the Windows Settings). This is done via Apply Teams (which `fleetctl` makes sure that it not only sets the profiles and labels associated with the team, but it also calls the batch-set profiles to upload those profiles). It would not make sense to support this via `Modify Team` (as nothing ensures that the profiles get uploaded).

Reply:
> this should be removed from the docs - the Modify team endpoint does not support setting the MacOS Settings (nor the Windows Settings)

@mna ah, ok! Thanks for catching that.

I fixed this in my latest commit (`e39e31d`). Please let me know if that change doesn't look right.

Reply:
Thanks @noahtalerman ! There are a few more lines to remove - the whole `macos_settings` and `windows_settings` sections (and sub-fields) should be removed, and I see that `enable_disk_encryption` is missing directly under `mdm`.

Reply:
I can make the change if you prefer, just let me know!

Reply:
@mna please feel free to make the changes! Thanks :)